src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -24,25 +24,43 @@ object Run  {
   val PollingBackoffFactor = 1.1
 
   def apply(pipeline: Pipeline): Run = {
-    val rpr = new RunPipelineRequest().setPipelineId(pipeline.id).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
     val tag = s"JES Run [UUID(${pipeline.workflow.shortId}):${pipeline.key.tag}]"
+    if (pipeline.pipelineId.isDefined == pipeline.runIdForResumption.isDefined) {
+      val message =
+        s"""
+          |$tag: Exactly one of JES pipeline ID or run ID for resumption must be specified to create a Run.
+          |pipelineId = ${pipeline.pipelineId}, runIdForResumption = ${pipeline.runIdForResumption}.
+        """.stripMargin
+      throw new RuntimeException(message)
+    }
+
+    def runPipeline: String = {
+
+      val rpr = new RunPipelineRequest().setPipelineId(pipeline.pipelineId.get).setProjectId(pipeline.projectId).setServiceAccount(JesServiceAccount)
 
-    rpr.setInputs(pipeline.jesParameters.collect({ case i: JesInput => i }).toRunMap)
-    Log.info(s"$tag Inputs:\n${stringifyMap(rpr.getInputs.asScala.toMap)}")
+      rpr.setInputs(pipeline.jesParameters.collect({ case i: JesInput => i }).toRunMap)
+      Log.info(s"$tag Inputs:\n${stringifyMap(rpr.getInputs.asScala.toMap)}")
 
-    rpr.setOutputs(pipeline.jesParameters.collect({ case i: JesOutput => i }).toRunMap)
-    Log.info(s"$tag Outputs:\n${stringifyMap(rpr.getOutputs.asScala.toMap)}")
+      rpr.setOutputs(pipeline.jesParameters.collect({ case i: JesOutput => i }).toRunMap)
+      Log.info(s"$tag Outputs:\n${stringifyMap(rpr.getOutputs.asScala.toMap)}")
 
-    val logging = new Logging()
-    logging.setGcsPath(pipeline.gcsPath)
-    rpr.setLogging(logging)
+      val logging = new Logging()
+      logging.setGcsPath(pipeline.gcsPath)
+      rpr.setLogging(logging)
```

Comment:
It might be better to do it the other way around since your changes are more straightforward, less risk of a merge hosing up the concurrency changes